services/mail_service.js

```diff
@@ -0,0 +1,57 @@
+const fetch = require("node-fetch")
+
+const serviceUrl = "https://email-service-estella.herokuapp.com/email"
+const selfUrl = "https://meeting-organizer-estella.herokuapp.com/"
```

Comment:
Jesteś pewny, że taki jest selfUrl? Bo wydaje mi się, że dawanie im dostępu do backu to trochę kiepski pomysł, a teoretycznie nie jest powiedziane, że to musi stać osobno. To musi być albo do frontu głównego jakiś link, albo w sumie nie wiem, jak inaczej.